pexpect/fdpexpect.py

```diff
@@ -112,3 +113,19 @@ def writelines(self, sequence):
         "Call self.write() for each item in sequence"
         for s in sequence:
             self.write(s)
+
+    def read_nonblocking(self, size=1, timeout=-1):
+        '''The read_nonblocking method of fdspawn assumes that the file
+        will never block. This is not the case for all file like objects
```

Comment:
Will this change fix the confusion:
`'''The read_nonblocking method of SpawnBase assumes that the file`